test/views/DeclarationInformationViewSpec.scala

```diff
@@ -19,14 +19,29 @@ package views
 import java.time.LocalDateTime
 
 import base.Injector
+import com.typesafe.config.{Config, ConfigFactory}
+import config.EADDocumentConfig
 import models.declaration.notifications.Notification
 import models.declaration.submissions.{Submission, SubmissionStatus}
+import play.api.Configuration
+import uk.gov.hmrc.govukfrontend.views.html.components.{GovukSummaryList, GovukTable}
 import views.declaration.spec.UnitViewSpec
+import views.html.components.gds.gdsMainTemplate
 import views.html.declaration_information
 
 class DeclarationInformationViewSpec extends UnitViewSpec with Injector {
 
-  private val declarationInformationPage = instanceOf[declaration_information]
+  private val gdsMainTemplate = instanceOf[gdsMainTemplate]
+  private val govukSummaryList = instanceOf[GovukSummaryList]
+  private val govukTable = instanceOf[GovukTable]
+
+  private val configWithEnabledEAD: Config =
+    ConfigFactory.parseString("microservice.services.features.ead-document=true")
+  private val configWithDisabledEAD: Config =
+    ConfigFactory.parseString("microservice.services.features.ead-document=false")
+
+  private val eadDocumentConfigEnabled = new EADDocumentConfig(Configuration(configWithEnabledEAD))
+  private val eadDocumentConfigDisabled = new EADDocumentConfig(Configuration(configWithDisabledEAD))
```

Comment:
Would it make sense to extract it to a separate class or trait? (depending if you consider inheritance or composition to be more applicable here)

Reply:
Hmm, not sure if it's necessary, this flag is just for a link, so the config for it shouldn't be used in different places